client/test/e2e/resolve.test.ts

```diff
@@ -1,6 +1,6 @@
 import { resolve } from '../../src';
 import { SolanaUtil } from '../../src/solana-util';
-import { ClusterType } from '../../src/solid-data';
+import { getClusterType } from '../../src/solid-data';
```

Comment:
I don't see where this function exists in the code -- should this be `ClusterType`?

Reply:
Fixed